docs/user/cassandra/usage.rst

```diff
@@ -1,8 +1,9 @@
 Using the Cassandra DataStore Programmatically
 ==============================================
 
-Since the Cassandra DataStore is just another Geotools DataStore, you can use it exactly as you would any other Geotools
-DataStore such as the PostGIS DataStore or the Accumulo DataStore.  To get a connection to a Cassandra DataStore, use the ```DataStoreFinder```.
+Since the Cassandra DataStore is just another Geotools DataStore, you can use it exactly as
+you would any other Geotools DataStore such as the PostGIS DataStore or the Accumulo DataStore.
```

Comment:
Also, maybe there's a way to rewrite this sentence so that it doesn't say "DataStore" 17 times?